### PR TITLE
updated powershell.exe help to reflect current PSCore6 behavior

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Reflects changes in PSCore6 and aligns with the built in help

Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [X] The documented feature was introduced in version (6) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/powershell_exe/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/powershell_exe/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C30%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201709302010193905-1707%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nPxMDNxC%2B9pbOU8nzsDEvcmanYNMjPTWTz9PBoiXWwQ%3D&st=2017-09-30T20%3A35%3A50Z&se=2017-10-01T20%3A45%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md [hunk 1]
```diff
@@ -42,6 +40,9 @@ PowerShell[.exe] -Help | -? | /?
 
 # PARAMETERS
 
+-Version
+Displayes the version of PowerShell.  Additional parameters
```
Comment (adityapatwardhan) [inline reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md]: Typo. Displayes -> Displays

Comment (SteveL-MSFT) [reply reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md]: Will fix

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/powershell_exe/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/powershell_exe/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201710021927455683-1707%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tDdrOdWfqfjBBpkshmBdbobe8MfJSSP8qeX%2BBwiacNc%3D&st=2017-10-02T19%3A53%3A16Z&se=2017-10-03T20%3A03%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**